pages/common/ps.md

```diff
@@ -13,3 +13,11 @@
 - Search for a process that matches a string:
 
 `ps aux | grep {{string}}`
+
+- List all processes of the current user in extra full format:
+
+`ps --user $(id --user) -F`
+
+- List all processes of the current user as ASCII art process hierarchy:
```

Comment:
How about changing it to - `List all processes of the current user as a tree:` ? And also, you can change `id --user` to `id -u` for brevity. As that is how its mentioned in the id.md page.

Reply:
I will update it to `as a tree`.

But will keep `id --user` as this is suggested in https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#user-content-guidelines 
> Try to incorporate the **spelled-out version** of single-letter options in the example's description.

Reply:
No you did not misread. There are several implicit rules so its not practical to mention every one of them. The addendum to this rule is, if the single-letter version is easily understandable, then use that. In this case, `-u` can be easily remembered for `user`. But if it was something like `-m|--user`, then `--user` would be preferred. 

Hope that makes sense.

Reply:
Ok. Now it's more clear to me.